app-emulation/fuse/fuse-1.3.8.ebuild

```diff
@@ -0,0 +1,64 @@
+# Copyright 1999-2017 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=6
+
+DESCRIPTION="Free Unix Spectrum Emulator by Philip Kendall"
+HOMEPAGE="http://fuse-emulator.sourceforge.net"
+SRC_URI="mirror://sourceforge/fuse-emulator/${P}.tar.gz"
+
+LICENSE="GPL-2"
+SLOT="0"
+KEYWORDS="~amd64 ~ppc ~x86"
+IUSE="alsa ao fbcon gpm gtk joystick memlimit png sdl svga X xml"
+
+# Only one UI back-end can be enabled at a time
+REQUIRED_USE="^^ ( X fbcon gtk sdl svga )"
+
+RDEPEND=">=app-emulation/libspectrum-1.3.5
+	dev-libs/glib:2
+	X? ( x11-libs/libX11
```

Comment:
Please use the same sort order in IUSE and remaining stuffs. I'd suggest not sorting uppercase before everything.